contribute/preinstalled-apps.rst

```diff
@@ -20,17 +20,17 @@ A full list of the Ubuntu Touch core apps follows.
 ================    ==================================  ============================
 Application Name    Store page                          Code repository
 ================    ==================================  ============================
-Calculator          `Calculator on OpenStore`_          `calculator-app on GitHub`_
+Calculator          `Calculator on OpenStore`_          `calculator-app on GitLab`_
 Calendar            `Calendar on OpenStore`_            `calendar-app on GitHub`_
 Clock               `Clock on OpenStore`_               `clock-app on GitHub`_
-File Manager        `File Manager on OpenStore`_        `filemanager-app on GitHub`_
-Gallery             `Gallery on OpenStore`_             `gallery-app on GitHub`_
-Music               Not yet in OpenStore                `music-app on GitHub`_
-Notes               `Notes on OpenStore`_               `notes-app on GitHub`_
-OpenStore           `OpenStore... on OpenStore`_        `openstore-app on GitHub`_
+File Manager        `File Manager on OpenStore`_        `filemanager-app on GitLab`_
+Gallery             `Gallery on OpenStore`_             `gallery-app on GitLab`_
+Music               `Music on OpenStore`_               `music-app on GitLab`_
+Notes               `Notes on OpenStore`_               `notes-app on GitLab`_
+OpenStore           `OpenStore... on OpenStore`_        `openstore-app on GitLab`_
 Terminal            `Terminal on OpenStore`_            `terminal-app on GitHub`_
-UBports Welcome     `UBports Welcome on OpenStore`_     `ubports-app on GitHub`_
-Weather             Not yet in OpenStore                `weather-app on GitHub`_
+UBports Welcome     `UBports Welcome on OpenStore`_     `ubports-app on GitLab`_
+Weather             `Weather on OpenStore`              `weather-app on GitLab`_
```

Comment:
You may have forgotten an underscore after the "Weather on OpenStore" link. This makes it italicized in the output without being a link.

Reply:
Oops, thanks for catching that!